acceptance/tests/utf8/utf8-in-file-resource.rb

```diff
@@ -0,0 +1,55 @@
+test_name 'utf-8 characters in resource title and param values' do
+  confine :except, :platform => [ 'windows', 'ubuntu-16']   # PUP-6983
+  utf8chars = "€‰ㄘ万竹ÜÖ"
```

Comment:
Yes, please do that. It is also helpful to know what the Unicode code points are so that they compared to bytes / other encodings / etc.